homework/index.js

```diff
@@ -33,14 +36,66 @@
   function main(url) {
     fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
+
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        //createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        const header = createAndAppend('header', root, { class: 'header' });
+        const h1 = createAndAppend('h1', header, { text: 'FooCoding Repos', class: 'h1' });
+
+        const select = createAndAppend('select', root, { class: 'select' });
+        createAndAppend('option', select, { text: 'Choose your favorite repo below:' });
+
+        data.forEach(repo => {
+          const name = repo.name;
+          createAndAppend('option', select, { text: name });
+          //console.log(name);
```

Comment:
Don't leave commented out code, remove it! It's easier for me to review and easier for you to read when you come back to it at a later time.